src/screens/internal/CompareSnapshots/CompareSnapshots.tsx

```diff
@@ -426,33 +428,29 @@ function useProjectionsSet(
         const topCounties = regions.topCountiesByPopulation(COUNTIES_LIMIT);
         setProjectionsSet(
           ProjectionsSet.fromProjections(
-            await fetchCountyProjections(leftSnapshot, topCounties),
-            await fetchCountyProjections(rightSnapshot, topCounties),
+            await fetchRegionProjections(leftSnapshot, topCounties),
+            await fetchRegionProjections(rightSnapshot, topCounties),
```

Comment:
We had separate `fetchCountyProjections()` and `fetchMetroProjections()` functions that were actually 100% equivalent. 😁